Maple2.Model/Enum/Gender.cs

```diff
@@ -1,7 +1,15 @@
-﻿namespace Maple2.Model.Enum;
+﻿using System;
+
+namespace Maple2.Model.Enum;
 
 public enum Gender : byte {
     Male = 0,
     Female = 1,
     All = 2,
 }
+
+[Flags]
+public enum GenderFlag : byte {
```

Comment:
To make it more clear that this is specifically for market filtering, I would move to another file and name it `GenderFilterFlag` or something. Same with `JobFlag` below. (`MeretMarket.cs` seems like a good choice, unless there are other places the enum is used)